exchanges/bybit/bybit_wrapper.go

```diff
@@ -1659,6 +1664,17 @@ func (by *Bybit) SetLeverage(ctx context.Context, item asset.Item, pair currency
 	}
 }
 
+// IsPerpetualFutureCurrency ensures a given asset and currency is a perpetual future
+func (by *Bybit) IsPerpetualFutureCurrency(a asset.Item, p currency.Pair) (bool, error) {
```

Comment:
Can drop error

Reply:
It is an interface function. Its definitely something I should reconsider, but I think it should be its own PR if I change it

Reply:
👍